### PR TITLE
EE-299: Auto add pr creator as jira ticket assignee

### PR DESCRIPTION
**Pull request by duanebester**

### CODE REVIEW

File: action.js [hunk 1]
```diff
@@ -196,4 +197,25 @@ module.exports = class {
...
+  async autoAssignCreator() {
+    const { user } = this.githubEvent.pull_request;
+    if (!user) {
+      throw new Error(`No User on PR? - ${user}`);
+    }
+
+    const rigupUser = await this.dynamo.findByGithubId(user.id);
+
+    if (!rigupUser) {
+      throw new Error(`PR by unknown user? - ${user}`);
```
Comment (macleanevans) [inline action.js]: What happens if we throw an error in a Github Action?

Comment (himichaelroberts) [reply action.js]: Would it be better to log this instead of throwing an error?

Comment (duanebester) [reply action.js]: @himichaelroberts ah yes, lemme change to log

Comment (duanebester) [reply action.js]: @macleanevans it gets caught in index.js :)

**Pull request merged**